Core/src/Report/MetadataProviderUtils.php

```diff
@@ -36,6 +36,9 @@ public static function autoSelect($server)
             }
             return new GAEFlexMetadataProvider($server);
         }
+        if (!empty(getenv('K_CONFIGURATION'))) {
+            return new CloudRunMetadataProvider(getenv());
```

Comment:
```suggestion
            return new CloudRunMetadataProvider(getenv('K_CONFIGURATION'));
```

Reply:
Good question. All three env vars are guaranteed to exist by Cloud Run (and others as well), so my decision was arbitrary

Reply:
The suggestion here won't work however, because the value of that env var is a string and does not contain the values we need that are in a separate env var